### DISCUSSION TITLE
How to use the flutter plugin

### DISCUSSION THREAD

**Discussion by User0918173** [singerdmx/flutter-quill#1944] [General]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/singerdmx/flutter-quill/issues)

### Flutter Quill version

Latest

### Steps to reproduce

Code is not working 

### Expected results

Code is not working 

### Actual results

Code is not working 

### Code sample

<details><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by EchoEllet** [comment]
Closing issue due to missing details, if you have any questions feel free to open a new discussion.
